### PR TITLE
Clarify inclusion of JavaScript

### PR DESCRIPTION
**Pull request by spoonerWeb**
* Set block "same as includeJS" to section "includeJSFooter" as well
* Add short descriptions of what the difference is between all "includeJSxxx" sections
* Move example in "includeJSFooter" to "includeJS"

### CODE REVIEW

File: Documentation/Setup/Page/Index.rst [hunk 1]
```diff
@@ -533,6 +533,10 @@ includeJS.[array]
                 # in the TS constants field.
                 file2 = javascript_uploaded_to_template*.js
                 file2.if.isTrue = {$myConstant}
+                
+                twitter = http://twitter.com/javascripts/blogger.js
```
Comment (mbrodala) [inline Documentation/Setup/Page/Index.rst]: This does not exist. And if, it should use HTTPS and probably a fixed version to ensure the mentioned `integrity` matches.

Comment (spoonerWeb) [reply Documentation/Setup/Page/Index.rst]: This was just moved from the doubled section I removed. Didn't check if this JS exists. What about using another JS like the latest of jQuery?

Comment (mbrodala) [reply Documentation/Setup/Page/Index.rst]: Good idea. :-)

Comment (spoonerWeb) [reply Documentation/Setup/Page/Index.rst]: Using jQuery now.

**Pull request merged**